### PR TITLE
Minor doc improvements

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Minor doc improvements

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [x] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [ ] Bump-up library or tool used for development (does not change the final image)
- [ ] CI configuration change
- [ ] Konflux configuration change

### CODE REVIEW

**Comment by tisnik** [comment]
/override "Red Hat Konflux / lightspeed-service-on-pull-request"
/approve

File: README.md [hunk 1]
```diff
@@ -500,6 +503,18 @@ OLS service can be started locally. In this case GradIO web UI is used to
 interact with the service. Alternatively the service can be accessed through
 REST API.
 
+[!TIP]
+To enable GradIO web UI you need to have following `dev_config` section in your configuration file:
```
Comment (JoaoFula) [inline README.md:507]: need to have **the** following

**Comment by tisnik** [comment]
/override "Red Hat Konflux / ols-enterprise-contract / lightspeed-service"

**Pull request merged**